python/test/unit/language/test_core_amd.py

```diff
@@ -1952,7 +1952,7 @@ def __str__(self):
         # MmaLayout(version=(2, 0), warps_per_cta=[1, 4]),
         # MmaLayout(version=1, warps_per_cta=[4, 1]),
         # MmaLayout(version=(2, 0), warps_per_cta=[4, 1]),
-        BlockedLayout([1, 4], [2, 32], [2, 2], [1, 0]),
+        BlockedLayout([1, 4], [2, 32], [2, 1], [1, 0]),
```

Comment:
After a second thought, I think it should be better
`BlockedLayout([1, 2], [2, 32], [2, 2], [1, 0])`
since the numWarps is set to 4, we'd better keep the total number of warps per CTA as 4.
Usually blocked encoding is computed from numWarps. But it is still mysterious to me why hard coding a different number of warps, or a different number of threads per warp, in the blocked encoding works.

Reply:
Thanks for your suggestion.  Done.